exercises/concept/high-school-sweetheart/.docs/introduction.md

```diff
@@ -0,0 +1,121 @@
+# String methods
```

Comment:
I would recommend using an introduction.md.tpl then using `configlet generate` to automate this file.